Makefile

```diff
@@ -60,11 +61,14 @@ docker-push:
 
 # docs regenerates flags in docs for all thanos commands.
 .PHONY: docs
-docs:
-	@go get -u github.com/campoy/embedmd
-	@go build ./cmd/thanos/...
```

Comment:
wonder why this was here there.. Now you removed that step totally (:

Reply:
sure but what about users who will want to build it using `make docs` (at actually this is the only way  to do it for them using Makefile)

Reply:
they still can do it, I haven't delete the `make docs` target, didn't understand the question https://github.com/improbable-eng/thanos/pull/441/files#diff-b67911656ef5d18c4ae36cb6741b7965R64